samples/apps-with-core-angular/tye.yaml

```diff
@@ -0,0 +1,19 @@
+# tye application configuration file
+# read all about it at https://github.com/dotnet/tye
+#
+# when you've given us a try, we'd love to know what you think:
+#    https://aka.ms/AA7q20u
+#
+name: project-tye
+services:
+- name: moviesapi
+  project: MoviesAPI/MoviesAPI.csproj
+  bindings:
+  - protocol: https
+    port: 5001
+  
+- name: moviesapp
+  image: moviesapp:dev
```

Comment:
For `tye run`, instead of specifying the image, could you specify the `dockerFile` as a relative path to the docker file? Curious to see if that works. This doesn't work for `tye deploy` yet, but it will soon.

Reply:
@jkotalik 
I have tested and added the change as default option tye file. However, in readme, I have explained both the scenarios with Dockerfile and with image. Thanks.